conf/ut/rconfc.c

```diff
@@ -87,7 +87,7 @@ static int rconfc_ut_motr_start(struct m0_rpc_machine    *mach,
 		"-c", M0_UT_PATH("diter.xc")
 	};
 	*rctx = (struct m0_rpc_server_ctx) {
-		.rsx_xprts         = &m0_conf_ut_xprt,
+		.rsx_xprts         = &xprt,
```

Comment:
Address of a local variable. Is this safe?

Reply:
incorporated